docs/development.rst

```diff
@@ -230,6 +230,23 @@ there).
 
 .. include:: ../CONTRIBUTING.rst
 
+
+Creating a release
+==================
+
+OpenSCM uses designated Github Actions to upload the package to PyPI
+(and, in the future, also to Conda). To create a release:
+
+1. Change the "master" header in ``CHANGELOG.rst`` to the release
+   version number (not starting with "v", e.g. "1.2.3") and create a
+   new, empyy "master" header above. Commit these changes.
```

Comment:
'Commit these changes.' --> 'Commit these changes with the message, 'Prepare for release of vVERSIONNUMBER'' e.g. 'Prepare for release of v1.2.3'